lib/elixir/lib/exception.ex

```diff
@@ -955,6 +955,18 @@ defmodule CompileError do
   end
 end
 
+defmodule Kernel.TypespecError do
+  defexception [:file, :line, description: "typespec error"]
```

Comment:
Do we need a default `description` here?

Reply:
I bet you one wouldn't hurt, let me add one

Reply:
@whatyouhide  I am not very sure we do, I am still trying to figure out why we had to change to this perhaps that would give me more context

Reply:
We always give a description, so we can remove this one:

```suggestion
  defexception [:file, :line, :description]
```